src/commands/postnsave.js

```diff
@@ -0,0 +1,154 @@
+// @flow
+
+import FeedParser from 'feedparser';
+import request from 'request';
+import fs from 'fs';
+
+import {owner} from '../config/config.json';
+
+import type {Message} from 'discord.js';
+import type {Stash} from '../type/Stash';
+
+let url = 'https://www.craneanime.com/wp/feed/atom';
+// let url = 'https://xkcd.com/atom.xml';
+
+export default {
+  name: 'postnsave',
+  description: 'Check the atom feed',
+  guildOnly: true,
+  execute(message: Message, args: String) {
+    // console.log('message', message.guild.id);
+    let newArticles = [];
+    if (message.author.id !== owner) {
+      return;
+    }
+
+    message.channel.send('Checking feed...').then((sentMessage) => {
+      if (Array.isArray(sentMessage)) {
```

Comment:
Why do we return if `sentMessage` is an array?

Reply:
this one comes from the ```check``` command. ill just remove it then sorry 😄

Reply:
Oops, that means I totally forgot about that.